src/component/TitleBar.tsx

```
@@ -30,34 +29,53 @@ export default function TitleBar() {
+                    <img
+                        src={minimizeButton}
+                        className="pointer-events-none"
+                        alt="minimize"
+                    />
                 </div>
-                {/* <div onClick={maxHandler}>
-                    max
-                </div> */}
-                <div onClick={closeHandler} className="hover:bg-accent-negative h-full flex items-center px-2 transition-colors">
-                    <img src={closeButton} className="pointer-events-none text-text-primary" alt="close" />
+                {import.meta.env.DEV && <div onClick={maxHandler}>max</div>}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)